package.json

```diff
@@ -73,12 +73,13 @@
     "create-react-context": "^0.2.1",
     "deepmerge": "^2.1.1",
     "flow-copy-source": "^2.0.2",
-    "hoist-non-react-statics": "^2.5.0",
+    "hoist-non-react-statics": "2.5.0",
```

Comment:
Can we not use the exact version? It'll prevent de-duplication by yarn/npm

Reply:
I tried but it didn't work. I still have an error "required module not found". So I decided to force 2.5.0 now to be able to release new version with other fixes